### ISSUE TITLE
"Full functionality" documentation for configuration language does not exist.

### ISSUE THREAD

**Issue by ian-bertolacci** [rubisco-sfa/ILAMB#20]
On the 'First Steps' tutorial, under 'Configure File' the [last paragraph](https://github.com/rubisco-sfa/ILAMB/blame/master/doc/first_steps.rst#L150) says: "The configuration language is small, but allows you to change a lot of the behavior of the system. The full functionality is documented **here.**" but the link ("**here**") points to "doc/nope_not_yet" , which I assume is/was a placeholder link.

Does the full functionality documentation for the configuration language exist somewhere else and this link was simply a placeholder?

**Comment by nocollier** [comment]
You are correct, I have never made a list of all the things ILAMB recognizes. For now I have pushed a correction in the docs. The next time I render then, it will update in the tutorial. For now the best I have to offer you is to look at the CMIP configure file that we distribute with the software:

https://github.com/rubisco-sfa/ILAMB/blob/master/src/ILAMB/data/cmip.cfg

Essentially, I made ILAMB recognize the things we needed it to. It is also extensible. If you are writing your own Confrontation, all keywords will get passed into a keywords dictionary which you can parse in the code. For example, in the permafrost work, we needed to be able to control the threshold for freezing temperature which can be passed via this configure langugage:

https://github.com/rubisco-sfa/ILAMB/blob/master/src/ILAMB/ConfPermafrost.py#L74

Let me know if you have a specific question and I am happy to answer.

**Comment by ian-bertolacci** [comment]
That makes sense. Is there any documentation about the `derived` algebraic language?
